corehq/apps/hqwebapp/urls.py

```diff
@@ -33,9 +33,10 @@
 domain_specific = patterns('corehq.apps.hqwebapp.views',
     url(r'^$', 'redirect_to_default', name='domain_homepage'),
     url(r'^login/$', 'domain_login', name='domain_login'),
-    url(r'^login/mobile/$', 'domain_login', name='domain_mobile_login', 
+    url(r'^login/mobile/$', 'domain_login', name='domain_mobile_login',
         kwargs={'template_name': 'login_and_password/mobile_login.html'}),
-    url(r'^retreive_download/(?P<download_id>[0-9a-fA-Z]{25,32})/$', 
-        'retrieve_download', {'template': 'style/includes/file_download.html' },
-        name='hq_soil_download')
+    url(r'^retreive_download/(?P<download_id>[0-9a-fA-Z]{25,32})/$',
```

Comment:
I think it probably doesn't matter (and I it looks like you didn't change it), but `retreive` is spelled incorrectly (it's spelled correctly on the next line)